manifests/client/configure.pp

```diff
@@ -13,6 +13,10 @@
 # $lbclient_cert_file_path - cert file path. Required only if $lbclient_use_cert_auth == true.
 # $lbclient_cert_password  - cert password. Required only if $lbclient_use_cert_auth == true.
 # $lbclient_cert_alias     - cert alias. Required only if $lbclient_use_cert_auth == true.
+# $use_physical_file       - It is generally assumed that the cert and ca files will bundled in the application binary,
```

Comment:
Not necessarily a deliberate decision on my part, but good catch. The 'lbclient...' fields correspond to parameters in the lightblue-client.properties file while this new one is a modifier to them, but not a property itself.

I have no strong preference here. I will submit a patch.